modules/apps/headless/headless-admin-workflow/headless-admin-workflow-test/src/testIntegration/java/com/liferay/headless/admin/workflow/resource/v1_0/test/WorkflowDefinitionResourceTest.java

```diff
@@ -53,6 +56,10 @@ public class WorkflowDefinitionResourceTest
 	public static void setUpClass() throws Exception {
 		BaseWorkflowDefinitionResourceTestCase.setUpClass();
 
+		PermissionThreadLocal.setPermissionChecker(
+			PermissionCheckerFactoryUtil.create(
+				UserLocalServiceUtil.getUser(TestPropsValues.getUserId())));
```

Comment:
Hey @pedro-oliveira446 can we use `PermissionCheckerMethodTestRule.INSTANCE` here?